client/cli/src/lib.rs

```diff
@@ -371,4 +372,33 @@ mod tests {
 			assert!(test_filter("telemetry", Level::TRACE));
```

Comment:
Maybe we can also add an example with dashes to this test 🤷

Reply:
Added